File: JavaToCSharpConverter/Output/RescueBlockAndUnitNumbersNOrMoreRule.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace RescueJ
{
public class RescueBlockAndUnitNumbersNOrMoreRule : RescueBlockAndUnitNumbersRule
{


  protected RescueBlockAndUnitNumbersNOrMoreRule(long ndxIn)
  {
    nativeNdx = ndxIn;
  }

  public RescueBlockAndUnitNumbersNOrMoreRule(int nBlocks)
  {
    nativeNdx = Create_RescueBlockAndUnitNumbersNOrMoreRule0(nBlocks);
  }

}

}

File: JavaToCSharpConverter/Source/RescueBlockAndUnitNumbersNOrMoreRule.java
package org.Rescue.rjni;
import java.lang.*;
/*****************************************************************

    RescueBlockAndUnitNumbersNOrMoreRule.java

    Rod Hanks   Aug 25 2005

******************************************************************/
public class RescueBlockAndUnitNumbersNOrMoreRule extends RescueBlockAndUnitNumbersRule
{


  protected RescueBlockAndUnitNumbersNOrMoreRule(long ndxIn)
  {
    nativeNdx = ndxIn;
  }

  public RescueBlockAndUnitNumbersNOrMoreRule(int nBlocks)
  {
    nativeNdx = Create_RescueBlockAndUnitNumbersNOrMoreRule0(nBlocks);
  }

  private native long Create_RescueBlockAndUnitNumbersNOrMoreRule0(int nBlocks);
}
